learners/setup.md

```diff
@@ -2,53 +2,136 @@
+[napari-aicsimagio](https://github.com/AllenCellModeling/napari-aicsimageio)
+ plugin to load different image types.
+For beginners we recommend using Anaconda to install and manage these
+packages. The first step is to
+[download and install Anaconda](https://www.anaconda.com/download#downloads).
 
-## Software Setup
+The remaining instructions are written assuming you will be installing and
+running the software from a terminal. To open a terminal follow the
+instructions for your operating system in the
+['Starting Conda' section of the Anaconda getting started pages](
+https://conda.io/projects/conda/en/latest/user-guide/getting-started.html#starting-conda).
+Run the following commands in the terminal (lines starting
+with # are comments and do not need to be run).
+You can run the commands by copy and pasting them into the terminal and
+pressing the <kbd>Enter</kbd>  key.
+
+```bash
+# Make sure conda is up to date
+conda update -n base conda
+
+# We need to use pip to install the napari-aicsimagio plugin, so configure
+# conda to work well with pip
+conda config --set pip_interop_enabled True
+
+# Create a virtual environment (napari-env) for the lessons
+conda create -y -n napari-env -c conda-forge python=3.9 bioformats_jar
+
+# Activate the napari-env virtual environment
+conda activate napari-env
+
+# Install napari and plugins using pip
+pip install "napari[all]" napari-aicsimageio
+```
+
+## Post Setup Checks
+
+Before starting the course it is important to check that your setup is working.
+
+::::::::::::::::::::::::::::::::::::::: challenge
+
+## Check napari opens a tiff image.
+
+Any napari installation should be able to open a tiff image. Start napari.
+Note that if you're already in the napari-env virtual environment from the
+installation steps it is not necessary to run `conda activate napari-env`.
+```bash
+conda activate napari-env
+napari
+```
+Open a file with
+
+`File > Open file(s)...`
 
-::::::::::::::::::::::::::::::::::::::: discussion
+- Navigate to the directory where you saved `00001_01.ome.tiff` earlier on.
+- Select `00001_01.ome.tiff` and click open.
+- If you see a `Choose reader` dialog, select `napari builtins`.
 
-### Details
+::::::::::::::::::::::::::
+::::::::::::::::::: solution
 
-Setup for different systems can be presented in dropdown menus via a `solution`
-tag. They will join to this discussion block, so you can give a general overview
-of the software used in this lesson here and fill out the individual operating
-systems (and potentially add more, e.g. online setup) in the solutions blocks.
+## Expected Output
 
-:::::::::::::::::::::::::::::::::::::::::::::::::::
+![If this is what you see your napari installation is working as expected.
+ If not then please check the installation or get in touch with the course
+ organisers for help.](fig/ome_00001.png){alt="A screenshot of freshly
+ opened 00001_01.ome.tiff" width='80%'}
 
-:::::::::::::::: solution
+Close napari: `File > exit`
```

Comment:
On Mac, I see `File > Close Window`

Reply:
Do you just get close window? I get both close window and exit (on linux). They seem to do the same thing.